.github/workflows/event.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Event Handler
+
+on:
+  check_suite:
+    types: [completed]
+  issue_comment:
+    types: [created]
+
+jobs:
+  event-handler:
+    name: Handle ${{ github.event_name }} ${{ github.event.action }} event
+    runs-on: ubuntu-latest
```

Comment:
We haven't thought much about this yet but do we want to have a rolling image? (i.e. -latest)? I know we don't do that for pipelines because we need repeatability but maybe it is OK for actions but updates could definitely break things.

Reply:
Good point. Or maybe keep a rolling image for tools only so we catch issues early?